rsocket/statemachine/RSocketStateMachine.cpp

```diff
@@ -88,6 +88,10 @@ void RSocketStateMachine::connectServer(
 bool RSocketStateMachine::resumeServer(
     yarpl::Reference<FrameTransport> frameTransport,
     const ResumeParameters& resumeParams) {
+  if (!isDisconnectedOrClosed()) {
+    disconnect(folly::exception_wrapper(
```

Comment:
Nit: You don't need the `folly::exception_wrapper()` here, the ctor is implicit.